src/components/Daos/DaosPage.tsx

```diff
@@ -245,15 +278,21 @@ const createSubscriptionObservable = (props: IStateProps, data: SubscriptionData
     }
     ${DAOFieldsFragment}
   `;
-  const memberOfDAOs = currentAccountAddress ? arc.getObservableList(memberDAOsquery, (r: any) => createDaoStateFromQuery(r.dao).dao, standardPolling()) : of([]);
-  // eslint-disable-next-line @typescript-eslint/camelcase
-  const followDAOs = followingDAOs.length ? arc.daos({ where: { id_in: followingDAOs }, orderBy: "name", orderDirection: "asc" }, standardPolling(true)) : of([]);
-
-  return combineLatest(
-    arc.daos({ orderBy: "name", orderDirection: "asc", first: PAGE_SIZE, skip: data ? data[0].length : 0 }, standardPolling(true)),
-    followDAOs,
-    memberOfDAOs
-  );
+
+
+
+  const arcs = getArcs();
+  const daosData = [];
+
+  for (const network in arcs) {
+    const arc = arcs[network];
+    daosData.push(arc.daos({ orderBy: "name", orderDirection: "asc", first: PAGE_SIZE, skip: data ? data[0].length : 0 }, standardPolling(true)));
+    // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
I suggest to specify this setting globally if we using is such way

Reply:
What setting exactly?

Reply:
`@typescript-eslint/camelcase`